src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -464,6 +464,83 @@ bool verify_reference_field(const std::string &protocol,
     return res;
 }
 
+bool verify_shapes_node(const Node &node, const Node &shape_map, Node &info)
+{
+  const std::string protocol = "mesh::topology::unstructured";
+  bool res = true;
+  res &= verify_integer_field(protocol, node, info);
+  
+  DataAccessor<int32> shapes(node.data_ptr(), node.dtype());
```

Comment:
FYI: You can simplify this pattern using:
`int32_accessor shapes = node.values();`

Reply:
Done in 52049eb